spec/spec_helper.rb

```diff
@@ -10,6 +10,7 @@
 require "minitest/autorun"
 require "minitest/spec"
 require "minitest/stub_const"
+require "pry"
```

Comment:
I suspect it may have been unintentional to commit this?

Reply:
Ops. Good catch. =)